packages/gatsby/gatsby-config.js

```diff
@@ -56,8 +56,8 @@ module.exports = {
       resolve: `gatsby-plugin-prefetch-google-fonts`,
       options: {
         fonts: [{
-          family: `Open Sans`,
-          variants: [`300`, `400`, `700`],
+          family: `Source Sans Pro`,
+          variants: [`300`, `400`, `600`, `700`],
```

Comment:
Let's keep the existing font

Reply:
I reverted the font, but kept the `600` variant. It was used in various places, but it wasn't working because it wasn't actually being included properly:

![image](https://user-images.githubusercontent.com/697707/95574674-0c8cd500-0a36-11eb-9288-056dcc1a9a5c.png)